.github/workflows/publish-courseware.yaml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker - learn
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish Tremor Courseware Docker image to Registry
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: tremorproject/tremor
```

Comment:
can we change this to a own image name that way we can tag it and make it clear this is not a production image and shouldn't be used there?